containersQa/testlib.bash

```diff
@@ -464,15 +464,30 @@ function onlyOneJdk() {
   javas=$(cat $(getOldLsLog) | cleanBashOutputs | grep java |wc -l)
   jres=$(cat $(getOldLsLog) | cleanBashOutputs | grep jre  |wc -l)
   #branch for jre vs jdk
-  if [ "$OTOOL_jresdk" == "jre"  ] ; then
-    echo "otool jresdk settings is: $OTOOL_jresdk"
-    # it is interval due to rhel 7/8 diffs
-    test $javas -ge 1 -a $javas -le 1
-    test $jres  -ge 5 -a $jres  -le 6
-  else # JDK
-    # it is interval due to rhel 7/8 diffs
-    test $javas -ge 5 -a $javas -le 6
-    test $jres  -ge 5 -a $jres  -le 6
+  # in JDK25+ parallel instalability feature was removed, resulting in less links
+  if [[ $OTOOL_JDK_VERSION -ge 25 ]]; then
+    if [ "$OTOOL_jresdk" == "jre"  ] ; then
+      echo "otool jresdk settings is: $OTOOL_jresdk"
+      # it is interval due to rhel 7/8 diffs
+      test $javas -ge 1 -a $javas -le 1
+      test $jres  -ge 4 -a $jres  -le 4
+    else # JDK
+      # it is interval due to rhel 7/8 diffs
```

Comment:
The  comemtn of rhel 7/8 is invalid. Remove it pls.
The ` echo "otool jresdk settings is: $OTOOL_jresdk"` is lsot under many circusmtances. Pls move it before main if